core/src/main/java/hudson/model/Slave.java

```diff
@@ -121,14 +126,14 @@
      * Whitespace-separated labels.
      */
     private String label="";
-    
+
     private /*almost final*/ DescribableList<NodeProperty<?>,NodePropertyDescriptor> nodeProperties = new DescribableList<NodeProperty<?>,NodePropertyDescriptor>(Jenkins.getInstance());
```

Comment:
I comment on other's whitespace changes iff they have been especially nit-picky in their own review comments